[
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/304425751",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/10463#issuecomment-304425751",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/10463",
    "id": 304425751,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNDQyNTc1MQ==",
    "user": {
      "login": "shaolinfry",
      "id": 19836615,
      "node_id": "MDQ6VXNlcjE5ODM2NjE1",
      "avatar_url": "https://avatars.githubusercontent.com/u/19836615?u=909b68b23f1426d01e5c102dcc00c83ee063d3be&v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/shaolinfry",
      "html_url": "https://github.com/shaolinfry",
      "followers_url": "https://api.github.com/users/shaolinfry/followers",
      "following_url": "https://api.github.com/users/shaolinfry/following{/other_user}",
      "gists_url": "https://api.github.com/users/shaolinfry/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/shaolinfry/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/shaolinfry/subscriptions",
      "organizations_url": "https://api.github.com/users/shaolinfry/orgs",
      "repos_url": "https://api.github.com/users/shaolinfry/repos",
      "events_url": "https://api.github.com/users/shaolinfry/events{/privacy}",
      "received_events_url": "https://api.github.com/users/shaolinfry/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-05-27T04:04:31Z",
    "updated_at": "2017-05-27T04:04:31Z",
    "author_association": "CONTRIBUTOR",
    "body": "Can someone restart the two failed jobs, they look like random Travis related fails.",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/304425751/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/304427504",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/10463#issuecomment-304427504",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/10463",
    "id": 304427504,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNDQyNzUwNA==",
    "user": {
      "login": "fanquake",
      "id": 863730,
      "node_id": "MDQ6VXNlcjg2MzczMA==",
      "avatar_url": "https://avatars.githubusercontent.com/u/863730?u=a0e0040aacd7d4f0787481e8ac30b494fa429e11&v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/fanquake",
      "html_url": "https://github.com/fanquake",
      "followers_url": "https://api.github.com/users/fanquake/followers",
      "following_url": "https://api.github.com/users/fanquake/following{/other_user}",
      "gists_url": "https://api.github.com/users/fanquake/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/fanquake/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
      "organizations_url": "https://api.github.com/users/fanquake/orgs",
      "repos_url": "https://api.github.com/users/fanquake/repos",
      "events_url": "https://api.github.com/users/fanquake/events{/privacy}",
      "received_events_url": "https://api.github.com/users/fanquake/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-05-27T04:51:57Z",
    "updated_at": "2017-05-27T05:35:39Z",
    "author_association": "MEMBER",
    "body": "Restarted both jobs.  For reference the errors were:\r\n```\r\nstdout:\r\n2017-05-27 01:32:40.308000 TestFramework (INFO): Initializing test directory /tmp/bitcoin_test_runner_20170527_013135/p2p-compactblocks_160\r\n2017-05-27 01:32:40.564000 TestFramework (ERROR): Assertion failed\r\nTraceback (most recent call last):\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 281, in start_nodes\r\n    rpcs.append(start_node(i, dirname, extra_args[i], rpchost, timewait=timewait, binary=binary[i]))\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 242, in start_node\r\n    wait_for_bitcoind_start(bitcoind_processes[i], url, i)\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 216, in wait_for_bitcoind_start\r\n    raise Exception('bitcoind exited with status %i during initialization' % process.returncode)\r\nException: bitcoind exited with status 1 during initialization\r\nDuring handling of the above exception, another exception occurred:\r\nTraceback (most recent call last):\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 146, in main\r\n    self.setup_network()\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 81, in setup_network\r\n    self.setup_nodes()\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 94, in setup_nodes\r\n    self.nodes = start_nodes(self.num_nodes, self.options.tmpdir, extra_args)\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 283, in start_nodes\r\n    stop_nodes(rpcs)\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 303, in stop_nodes\r\n    assert not bitcoind_processes.values() # All connections must be gone now\r\nAssertionError\r\n2017-05-27 01:32:40.565000 TestFramework (INFO): Stopping nodes\r\nstderr:\r\n************************\r\nEXCEPTION: St12out_of_range       \r\nmap::at       \r\nbitcoin in AppInit()       \r\nTraceback (most recent call last):\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/p2p-compactblocks.py\", line 945, in <module>\r\n    CompactBlocksTest().main()\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 162, in main\r\n    self.stop_nodes()\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 206, in stop_nodes\r\n    stop_nodes(self.nodes)\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 301, in stop_nodes\r\n    for i, node in enumerate(nodes):\r\nTypeError: 'NoneType' object is not iterable\r\n.......................................................\r\np2p-segwit.py failed, Duration: 5 s\r\nstdout:\r\n2017-05-27 01:33:04.891000 TestFramework (INFO): Initializing test directory /tmp/bitcoin_test_runner_20170527_013135/p2p-segwit_156\r\n2017-05-27 01:33:10.047000 TestFramework (ERROR): Assertion failed\r\nTraceback (most recent call last):\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 281, in start_nodes\r\n    rpcs.append(start_node(i, dirname, extra_args[i], rpchost, timewait=timewait, binary=binary[i]))\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 242, in start_node\r\n    wait_for_bitcoind_start(bitcoind_processes[i], url, i)\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 216, in wait_for_bitcoind_start\r\n    raise Exception('bitcoind exited with status %i during initialization' % process.returncode)\r\nException: bitcoind exited with status 1 during initialization\r\nDuring handling of the above exception, another exception occurred:\r\nTraceback (most recent call last):\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 146, in main\r\n    self.setup_network()\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/p2p-segwit.py\", line 120, in setup_network\r\n    self.setup_nodes()\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 94, in setup_nodes\r\n    self.nodes = start_nodes(self.num_nodes, self.options.tmpdir, extra_args)\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 283, in start_nodes\r\n    stop_nodes(rpcs)\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 303, in stop_nodes\r\n    assert not bitcoind_processes.values() # All connections must be gone now\r\nAssertionError\r\n2017-05-27 01:33:10.048000 TestFramework (INFO): Stopping nodes\r\nstderr:\r\n************************\r\nEXCEPTION: St12out_of_range       \r\nmap::at       \r\nbitcoin in AppInit()       \r\nTraceback (most recent call last):\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/p2p-segwit.py\", line 1952, in <module>\r\n    SegWitTest().main()\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 162, in main\r\n    self.stop_nodes()\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 206, in stop_nodes\r\n    stop_nodes(self.nodes)\r\n  File \"/home/travis/build/bitcoin/bitcoin/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 301, in stop_nodes\r\n    for i, node in enumerate(nodes):\r\nTypeError: 'NoneType' object is not iterable\r\n```",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/304427504/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/304949034",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/10463#issuecomment-304949034",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/10463",
    "id": 304949034,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNDk0OTAzNA==",
    "user": {
      "login": "jtimon",
      "id": 1008458,
      "node_id": "MDQ6VXNlcjEwMDg0NTg=",
      "avatar_url": "https://avatars.githubusercontent.com/u/1008458?u=636a89c913ac407f52bb8d57c4fff384b98f421f&v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/jtimon",
      "html_url": "https://github.com/jtimon",
      "followers_url": "https://api.github.com/users/jtimon/followers",
      "following_url": "https://api.github.com/users/jtimon/following{/other_user}",
      "gists_url": "https://api.github.com/users/jtimon/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/jtimon/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/jtimon/subscriptions",
      "organizations_url": "https://api.github.com/users/jtimon/orgs",
      "repos_url": "https://api.github.com/users/jtimon/repos",
      "events_url": "https://api.github.com/users/jtimon/events{/privacy}",
      "received_events_url": "https://api.github.com/users/jtimon/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-05-30T17:23:15Z",
    "updated_at": "2017-05-30T17:23:15Z",
    "author_association": "MEMBER",
    "body": "Sorry, the error was just one s/-bip9params/-vbparams/ lost in rebase, not a problem with travis. Updated.",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/304949034/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305095347",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/10463#issuecomment-305095347",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/10463",
    "id": 305095347,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNTA5NTM0Nw==",
    "user": {
      "login": "shaolinfry",
      "id": 19836615,
      "node_id": "MDQ6VXNlcjE5ODM2NjE1",
      "avatar_url": "https://avatars.githubusercontent.com/u/19836615?u=909b68b23f1426d01e5c102dcc00c83ee063d3be&v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/shaolinfry",
      "html_url": "https://github.com/shaolinfry",
      "followers_url": "https://api.github.com/users/shaolinfry/followers",
      "following_url": "https://api.github.com/users/shaolinfry/following{/other_user}",
      "gists_url": "https://api.github.com/users/shaolinfry/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/shaolinfry/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/shaolinfry/subscriptions",
      "organizations_url": "https://api.github.com/users/shaolinfry/orgs",
      "repos_url": "https://api.github.com/users/shaolinfry/repos",
      "events_url": "https://api.github.com/users/shaolinfry/events{/privacy}",
      "received_events_url": "https://api.github.com/users/shaolinfry/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-05-31T06:21:49Z",
    "updated_at": "2017-05-31T06:21:49Z",
    "author_association": "CONTRIBUTOR",
    "body": "@fanquake can you rerun Travis - looks like the VM errored and stopped the build process prematurely.\r\n\r\nutACK assuming they pass.",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305095347/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305116538",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/10463#issuecomment-305116538",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/10463",
    "id": 305116538,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNTExNjUzOA==",
    "user": {
      "login": "fanquake",
      "id": 863730,
      "node_id": "MDQ6VXNlcjg2MzczMA==",
      "avatar_url": "https://avatars.githubusercontent.com/u/863730?u=a0e0040aacd7d4f0787481e8ac30b494fa429e11&v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/fanquake",
      "html_url": "https://github.com/fanquake",
      "followers_url": "https://api.github.com/users/fanquake/followers",
      "following_url": "https://api.github.com/users/fanquake/following{/other_user}",
      "gists_url": "https://api.github.com/users/fanquake/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/fanquake/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
      "organizations_url": "https://api.github.com/users/fanquake/orgs",
      "repos_url": "https://api.github.com/users/fanquake/repos",
      "events_url": "https://api.github.com/users/fanquake/events{/privacy}",
      "received_events_url": "https://api.github.com/users/fanquake/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-05-31T08:05:11Z",
    "updated_at": "2017-05-31T08:05:11Z",
    "author_association": "MEMBER",
    "body": "@shaolinfry Restarted, looks like they are all passing now.",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305116538/reactions",
      "total_count": 1,
      "+1": 1,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305209217",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/10463#issuecomment-305209217",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/10463",
    "id": 305209217,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNTIwOTIxNw==",
    "user": {
      "login": "shaolinfry",
      "id": 19836615,
      "node_id": "MDQ6VXNlcjE5ODM2NjE1",
      "avatar_url": "https://avatars.githubusercontent.com/u/19836615?u=909b68b23f1426d01e5c102dcc00c83ee063d3be&v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/shaolinfry",
      "html_url": "https://github.com/shaolinfry",
      "followers_url": "https://api.github.com/users/shaolinfry/followers",
      "following_url": "https://api.github.com/users/shaolinfry/following{/other_user}",
      "gists_url": "https://api.github.com/users/shaolinfry/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/shaolinfry/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/shaolinfry/subscriptions",
      "organizations_url": "https://api.github.com/users/shaolinfry/orgs",
      "repos_url": "https://api.github.com/users/shaolinfry/repos",
      "events_url": "https://api.github.com/users/shaolinfry/events{/privacy}",
      "received_events_url": "https://api.github.com/users/shaolinfry/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-05-31T14:45:36Z",
    "updated_at": "2017-05-31T14:45:36Z",
    "author_association": "CONTRIBUTOR",
    "body": "utACK b463bc9\r\n",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305209217/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305442256",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/10463#issuecomment-305442256",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/10463",
    "id": 305442256,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNTQ0MjI1Ng==",
    "user": {
      "login": "laanwj",
      "id": 126646,
      "node_id": "MDQ6VXNlcjEyNjY0Ng==",
      "avatar_url": "https://avatars.githubusercontent.com/u/126646?u=13b19d1ff2f5f914e180c41418f451a4ba6f8bd1&v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/laanwj",
      "html_url": "https://github.com/laanwj",
      "followers_url": "https://api.github.com/users/laanwj/followers",
      "following_url": "https://api.github.com/users/laanwj/following{/other_user}",
      "gists_url": "https://api.github.com/users/laanwj/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/laanwj/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/laanwj/subscriptions",
      "organizations_url": "https://api.github.com/users/laanwj/orgs",
      "repos_url": "https://api.github.com/users/laanwj/repos",
      "events_url": "https://api.github.com/users/laanwj/events{/privacy}",
      "received_events_url": "https://api.github.com/users/laanwj/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-06-01T09:35:55Z",
    "updated_at": "2017-06-01T09:35:55Z",
    "author_association": "MEMBER",
    "body": "Concept ACK, I think this naming is better because it is more general.",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305442256/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305467232",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/10463#issuecomment-305467232",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/10463",
    "id": 305467232,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNTQ2NzIzMg==",
    "user": {
      "login": "benma",
      "id": 1225267,
      "node_id": "MDQ6VXNlcjEyMjUyNjc=",
      "avatar_url": "https://avatars.githubusercontent.com/u/1225267?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/benma",
      "html_url": "https://github.com/benma",
      "followers_url": "https://api.github.com/users/benma/followers",
      "following_url": "https://api.github.com/users/benma/following{/other_user}",
      "gists_url": "https://api.github.com/users/benma/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/benma/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/benma/subscriptions",
      "organizations_url": "https://api.github.com/users/benma/orgs",
      "repos_url": "https://api.github.com/users/benma/repos",
      "events_url": "https://api.github.com/users/benma/events{/privacy}",
      "received_events_url": "https://api.github.com/users/benma/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-06-01T11:32:19Z",
    "updated_at": "2017-06-01T11:32:19Z",
    "author_association": "NONE",
    "body": "utACK b463bc9",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305467232/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305768842",
    "html_url": "https://github.com/bitcoin/bitcoin/pull/10463#issuecomment-305768842",
    "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/10463",
    "id": 305768842,
    "node_id": "MDEyOklzc3VlQ29tbWVudDMwNTc2ODg0Mg==",
    "user": {
      "login": "instagibbs",
      "id": 5767891,
      "node_id": "MDQ6VXNlcjU3Njc4OTE=",
      "avatar_url": "https://avatars.githubusercontent.com/u/5767891?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/instagibbs",
      "html_url": "https://github.com/instagibbs",
      "followers_url": "https://api.github.com/users/instagibbs/followers",
      "following_url": "https://api.github.com/users/instagibbs/following{/other_user}",
      "gists_url": "https://api.github.com/users/instagibbs/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/instagibbs/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/instagibbs/subscriptions",
      "organizations_url": "https://api.github.com/users/instagibbs/orgs",
      "repos_url": "https://api.github.com/users/instagibbs/repos",
      "events_url": "https://api.github.com/users/instagibbs/events{/privacy}",
      "received_events_url": "https://api.github.com/users/instagibbs/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-06-02T12:11:23Z",
    "updated_at": "2017-06-02T12:11:23Z",
    "author_association": "MEMBER",
    "body": "utACK https://github.com/bitcoin/bitcoin/pull/10463/commits/b463bc9215192e6fd93336fdca51778e9aa23d64",
    "reactions": {
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/305768842/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  }
]